app/src/main/java/ru/firstSet/kotlinOne/FragmentMoviesList.kt

```diff
@@ -0,0 +1,50 @@
+package ru.firstSet.kotlinOne
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.Fragment
+
+class FragmentMoviesList() : Fragment() {
+    private var someFragmentClickListener: SomeFragmentClickListener? = null
+    private var fmlCombinedShape: View? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? = inflater.inflate(R.layout.fragment_movies_list, container, false)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        fmlCombinedShape = view.findViewById<View>(R.id.fmlCombinedShape).apply {
+            setOnClickListener { someFragmentClickListener?.onChangeFragment() }
```

Comment:
@kamar19 Please add a method to this fragment with direct access to `supportFragmentManager` using `activity`
Better to use listener here to pass data to the second fragment (for example id of a list item in this case) it will help you in future homeworks

Reply:
Вот тут я не понял, что нужно сделать.
Мы еще не использовали списки, может отставим это фрагмент на будущее?

Reply:
@kamar19 Up to you!